docs/x_509.md

```diff
@@ -1,6 +1,6 @@
 # Creating and using an X.509 certificate
 
-You can test creating your own manifests using the pre-built certificates in the [sample folder](https://github.com/contentauth/c2patool/tree/main/sample). To use your own generated certificates, specify the path to the cert files in the following manifest fields:
+You can test creating your own manifests using the pre-built certificates in the [c2patool repo sample folder](https://github.com/contentauth/c2patool/tree/main/sample). To use your own generated certificates, specify the path to the certificate files in the following manifest fields:
```

Comment:
As I mentioned, you don't need to use those certs, since they are already embedded as defaults in the c2patool binary. They are there mostly as examples for what the certs should look like.